dev/benchmarks/complex_layout/test_driver/scroll_perf_test.dart

```diff
@@ -14,8 +14,7 @@ void main() {
     setUpAll(() async {
       driver = await FlutterDriver.connect();
 
-      // TODO(liyuqian): enable the following once it's proved to be non-flaky by transition_perf_test.dart.
-      // await driver.waitUntilFirstFrameRasterized();
+       await driver.waitUntilFirstFrameRasterized();
```

Comment:
nit: Is there an extra space in front of the await?

Reply:
Good catch! Fixed :)